src/App.tsx

```diff
@@ -1,26 +1,114 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos, USER_ID, deleteTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterTypes } from './types/FilterTypes';
+import classNames from 'classnames';
+import { filterTodos } from './helper/utilsFunctions';
+import { Footer } from './components/Footer/Footer';
+import { ErrorMessage } from './types/ErrorMessage';
+/* eslint-disable-next-line max-len */
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
```

Comment:
```suggestion
import { 
  ErrorNotification
 } from './components/ErrorNotification/ErrorNotification';
```
disable linter it's a bad practice